pkg/lib/cockpit-components-table.scss

```diff
@@ -1,6 +1,4 @@
 @import "global-variables";
-@import "@patternfly/patternfly/components/Table/table.scss";
-@import "@patternfly/patternfly/components/Table/table-grid.scss";
```

Comment:
It's not clear to me how the "null loader" is related to the scss includes here. Could you please explain this a little better?

Reply:
oh right. https://github.com/cockpit-project/cockpit/blob/main/webpack.config.js#L492..L504 

table, toolbar and page css files were passed to null-loader before, therefore not auto imported when these react components were used. No I changed the approach and I don't use null-loader., but I fix the breakpoints with inline replacement of the problematic numbers. So we don't need to manually import these scss file for Table, Tooltip and Page anymore.